Standards/scs-XXXX-v1-security-of-iaas-service-software.md

```diff
@@ -0,0 +1,138 @@
+
+This would avoid to have conflicting versions of software in terms of feature parity, while also allowing older software.
+Using this approach requires an additional advise to CSPs to update or implement patches for security issues.
+
+#### Advise CSPs to integrate software updates
+
+As long as maintained versions of software are used, updates with security patches are available and only need to be integrated.
+This can and should be done in a reasonable short timeframe.
+
+But CSPs may even use releases of IaaS software, that are either not maintained anymore by an open source community or may be even closed source implementations of the mandatory IaaS APIs.
+Allowing older versions or closed source software would only be acceptable, when CSPs assure (e.g. in documentation), that they themself will patch the software within their deployments.
+Security bug fixes must be implemented and proof of the fix then provided.
+Only under these circumstances deployments with older or alternative IaaS Layer software may be handled as compliant.
+
+This option could be taken for granted, but to actually advise using it may encourage CSPs to take a closer look on their life-cycle management and security risk handling.
+And CSPs using OpenStack could even be encouraged to upgrade their deployments.
+
+#### Dependencies of the IaaS Layer Software
+
+While the IaaS service software like OpenStack itself is monitored and security issues announced in OSSNs and OSSAs, these services have lots of dependecies, that are not monitored by the same entity.
+When dependencies have security issues, there might be no OSSN or OSSA, so CSPs also need to watch CVEs concerning these dependencies themselves.
+Those dependencies must also be updated in a reasonable timeframe, when a security issue is disclosed.
+
+#### What timeframe is needed to fix the issue?
+
+CSPs should be encouraged to fix security issues as fast as possible.
+Some security issues are very easy to exploit so as soon as the vulnerability is disclosed attacks on deployments will start.
+Other vulnerabilities may need much knowledge and more time to be exploited.
+Also the impact of different vulnerabilities will differ.
+
+So it can be concluded that some security issues need to be fixed immediately while for others it is okay to take some time.
+The BSI already has some guidance[^1] on how fast CSPs should respond.
+From the moment a vulnerability is disclosed these are the advised reaction times ranked by the severity of the vulnerability:
+
+1. Critical (CVSS = 9.0 – 10.0): 3 hours
+2. High (CVSS = 7.0 – 8.9): 3 days
+3. Mid (CVSS = 4.0 – 6.9): 1 month
+4. Low (CVSS = 0.1 – 3.9): 3 months
+
+[^1]: [C5 criteria catalog with timeframes for responses on page 75.](https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Publikationen/Broschueren/C5_2020.pdf?__blob=publicationFile&v=3)
```

Comment:
```suggestion
[^1]: [C5 criteria catalog with timeframes for responses on page 70.](https://www.bsi.bund.de/SharedDocs/Downloads/EN/BSI/CloudComputing/ComplianceControlsCatalogue/2020/C5_2020.pdf?__blob=publicationFile&v=3)
```

There seems to be an english version of that document that also contains the referenced paragraph.